beets/autotag/mb.py

```diff
@@ -217,6 +217,10 @@ def album_info(release):
         format = medium.get('format')
 
         all_tracks = medium['track-list']
+        if 'data-track-list' in medium:
+            all_tracks += medium['data-track-list']
+        total = len(all_tracks)
```

Comment:
IMHO, `total` is not a very good name. I would probably just remove that line and…

Reply:
Perhaps something like `track_count` would be more descriptive?